README.rst

```diff
@@ -71,13 +71,40 @@ long-term compatibility with projects conflicts partially with providing and usi
 the latest Python versions bring.
 
 Since we pour more `effort into maintaining and developing icalendar <https://github.com/collective/icalendar/discussions/360>`__,
-we split the project into two:
+this is an overview of the versions:
 
-- `Branch 4.x <https://github.com/collective/icalendar/tree/4.x>`__ with maximum compatibility to Python versions ``2.7`` and ``3.4+``, ``PyPy2`` and ``PyPy3``.
-- `Branch master <https://github.com/collective/icalendar/>`__ with the compatibility to Python versions ``3.7+`` and ``PyPy3``.
+Version 6
+~~~~~~~~~
 
-We expect the ``master`` branch with versions ``5+`` receive the latest updates and features,
-and the ``4.x`` branch the subset of security and bug fixes only.
+Version 6 of ``icalendar`` switches the timezone implementation to ``zoneinfo``.
+
+    >>> dt = icalendar.Calendar.example("timezoned").walk("VEVENT")[0]["DTSTART"].dt
+    >>> dt.tzinfo
+    ZoneInfo(key='Europe/Vienna')
+
+If you would like to continue to use ``pytz`` and receive the latest updates, you
+can switch back:
+
+    >>> icalendar.use_pytz()
+    >>> dt = icalendar.Calendar.example("timezoned").walk("VEVENT")[0]["DTSTART"].dt
+    >>> dt.tzinfo
+    <DstTzInfo 'Europe/Vienna' CET+1:00:00 STD>
+
+Version 6 is on `branch master <https://github.com/collective/icalendar/>`_ with compatibility to Python versions ``3.7+`` and ``PyPy3``.
+We expect the ``master`` branch with versions ``6+`` to receive the latest updates and features.
+
+Version 5
+~~~~~~~~~
+
+Version 5 uses only the ``pytz`` timezone implementation, and not ``zoneinfo``.
+No updates will be released for this.
+Please use version 6 and switch to use ``zoneinfo`` as documented above.
+
+Version 4
+~~~~~~~~~
+
+Version 4 is on `branch 4.x <https://github.com/collective/icalendar/tree/4.x>`_ with maximum compatibility with Python versions ``2.7`` and ``3.4+``, ``PyPy2`` and ``PyPy3``.
+The ``4.x`` branch only receives security and bug fixes if someone makes the effort.
```

Comment:
Looking at these:
- https://github.com/collective/icalendar/issues/513
- https://github.com/collective/icalendar/issues/459

We do not really make a release for 4.x, nobody invests time. For 5.x there will not really be a future release either because you can add one line to the code and then use 6.x. So, the claims will be dropped to support older versions but this is not part of this PR. The PR is for zoneinfo compatibility, also not for the next release. We can create an issue for the next release and what should be done, I think.